processing.js

```diff
@@ -237,6 +237,9 @@ module.exports = {
     SPHERE:         40,
     BOX:            41,
 
+    // Arc mode
+    CHORD:          2,
```

Comment:
we can at least add the OPEN and PIE constants here, even if the OPEN constant has no effect, and the PIE rendering is not imlemented yet.